CHARTER.md

```diff
@@ -309,6 +310,47 @@ o. Any issues that cannot be resolved by the TOB shall be referred to The Linux
    Foundation Executive Director for mediation, and, if necessary, for
    resolution by The Linux Foundation Board of Directors.
 
+p. OCI Working Groups.
+
+   - i. The OCI may establish one or more working groups (each, an “OCI Working
+     Group”) to experiment, validate, and define new OCI Specifications or
+     major revisions to existing OCI Specifications. Each OCI Working Group
+     shall have a defined scope of the OCI Working Group's intended activities
+     and goals. The end result of the OCI Working Group's work product shall be
+     either a new OCI Specification, an update to an existing OCI
+     Specification, or a new OCI Project.
+
+   - ii. Any participant in the OCI technical community may propose a working
+     group to the TOB for consideration. The TOB shall maintain a public list
+     of the information required to be included in a working group proposal.
+
+   - iii. Approval of a new OCI Working Group requires a two-thirds vote of the
+     TOB. The TOB will maintain a public list of all approved OCI Working
+     Groups.
+
+   - iv. Once an OCI Working Group determines that it has achieved its
+     objectives, the OCI Working Group leaders shall present to the TOB the
+     draft specification or project, along with such other information as the
+     TOB may require for evaluation of the draft specification or project. The
+     TOB shall maintain a public list of such required information.
+
+   - v. Approval of the draft specification (to be released as an OCI
+     Specification) or draft project (to become an OCI Project) shall require a
+     two-thirds vote of the TOB. For a draft specification, prior to such
+     approval the OCI Working Group may not describe it as a released OCI
+     Specification, and must designate it with a `-draft.#` suffix.
+
+   - vi. Following TOB approval of a draft specification to be released as an
```

Comment:
The implication here is that someone that submits a draft that is approved becomes the maintainers (which is a commitment, so maybe it should be stated).

Reply:
That was not actually the point and I think could be called out better. The original idea is that the working group would have owners, and when a draft is approved and becomes a specification, the set of maintainers is voted on at that time as well. Becoming a maintainer is a commitment, but there should be no requirement that owners are also the maintainers. Owners are responsible for finding maintainers though before a draft could be approved as a new specification/project. I commented below about disambiguating owners and maintainers.

Reply:
@vsoch and @dmcgowan, good point. In the initial draft I hadn't fully appreciated some of the intended distinctions between owners and maintainers. In the follow-up commit I've rephrased some of the details here in the charter and in the WG governance documents. Take a look and let me know if you are good with the changes, or if there are other specific changes you'd like to see.